packages/vats/test/test-lib-chainStorage.js

```diff
@@ -32,23 +32,20 @@ test('makeChainStorageRoot', async t => {
   t.is(rootNode.getKey(), rootKey, 'root key matches initialization input');
 
   // Values must be strings.
-  const nonStrings = new Map([
-    ['number', 1],
-    ['bigint', 1n],
-    ['boolean', true],
-    ['null', null],
-    ['undefined', undefined],
-    ['symbol', Symbol('foo')],
-    ['array', ['foo']],
-    [
-      'object',
-      {
-        toString() {
-          return 'foo';
-        },
-      },
-    ],
-  ]);
+  /** @type { Map<string, any> } */
+  const nonStrings = new Map();
+  nonStrings.set('number', 1);
```

Comment:
why the change from `new Map(...contents...)` to using `.set()`?

Reply:
sigh.

Reply:
Ah, but `new Map(Object.entries(…))` works: 4afe517aeb0b14b5702d8a83c3b76157ec04c7b0
https://github.com/Agoric/agoric-sdk/blob/4afe517aeb0b14b5702d8a83c3b76157ec04c7b0/packages/vats/test/test-lib-chainStorage.js#L35-L50